ocaml/xapi/xapi_diagnostics.ml

```diff
@@ -0,0 +1,18 @@
+module D = Debug.Make(struct let name="xapi_diagnostics" end)
+open D
+
+let gc_compact ~__context ~session ~host =
+  raise Api_errors.(Server_error(not_implemented, [ "gc_compact" ]))
```

Comment:
Why not implemented?

Reply:
hmm, this is an outdated line. @liulinC was adding a Class at first and then add the messages. It's here to make it build